### PR TITLE
Chores/backport k8s

### PR DESCRIPTION
**Pull request by armandfardeau**
#### :tophat: Description
*Please describe your pull request.*

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #?
- Fixes #?
- [Notion card]()

#### Testing
*Describe the best way to test or validate your PR.*

Example:
* Log in as admin
* Access Backoffice
* Go to organization settings
* See ...

#### Tasks
- [ ] Add specs
- [ ] Add note about overrides in OVERLOADS.md
- [ ] In case of new dependencies or version bump, update related documentation

### :camera: Screenshots
*Please add screenshots of the changes you're proposing if related to the UI*

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,14 +1,48 @@
+GIT
```
Comment (Quentinchampenois) [inline Gemfile.lock:1]: Rebuild Gemfile.lock

File: config/initializers/decidim.rb [hunk 2]
```diff
@@ -1,19 +1,29 @@
...
-  config.skip_first_login_authorization = ENV["SKIP_FIRST_LOGIN_AUTHORIZATION"] ? ActiveRecord::Type::Boolean.new.cast(ENV["SKIP_FIRST_LOGIN_AUTHORIZATION"]) : true
   config.application_name = "OSP Agora"
   config.mailer_sender = "OSP Agora <ne-pas-repondre@opensourcepolitics.eu>"
 
   # Change these lines to set your preferred locales
-  config.default_locale = :en
-  config.available_locales = [:en, :fr]
+  if Rails.env.production?
+    config.default_locale = ENV.fetch("DEFAULT_LOCALE", "fr").to_sym
+    config.available_locales = ENV.fetch("AVAILABLE_LOCALES", "fr").split(",").map(&:to_sym)
```
Comment (Quentinchampenois) [inline config/initializers/decidim.rb:13]:
```suggestion
    config.available_locales = ENV.fetch("AVAILABLE_LOCALES", "en,fr").split(",").map(&:to_sym)
```

File: config/initializers/decidim.rb [hunk 3]
```diff
@@ -24,22 +34,14 @@
     }
   }
```
Comment (Quentinchampenois) [inline config/initializers/decidim.rb:36]: Add Decidim::Initiatives block

File: config/initializers/decidim.rb [hunk 4]
```diff
@@ -97,9 +99,42 @@
   end
 
   config.base_uploads_path = "#{ENV["HEROKU_APP_NAME"]}/" if ENV["HEROKU_APP_NAME"].present?
+
+  # Machine Translation Configuration
+  #
+  # Enable machine translations
+  config.enable_machine_translations = Rails.application.secrets.translator[:enabled]
```
Comment (Quentinchampenois) [inline config/initializers/decidim.rb:106]: Remove machine translations

File: config/initializers/decidim.rb [hunk 5]
```diff
@@ -97,9 +99,42 @@
...
+  config.machine_translation_service = "DeeplTranslator"
+  config.machine_translation_delay = Rails.application.secrets.translator[:delay]
+end
+
+Decidim.module_eval do
```
Comment (Quentinchampenois) [inline config/initializers/decidim.rb:111]: Remove reminder registry

File: config/locales/fr.yml [hunk 6]
```diff
@@ -102,21 +246,10 @@ fr:
...
-      user_menu:
-        account: Mon compte
-        admin_dashboard: Tableau de bord administrateur
-        conversations: Conversations
-        notifications: Notifications
-        profile: Mon compte
-        public_profile: Mon profil public
-        sign_out: Déconnexion
-        title: Titre
+  rack_attack:
```
Comment (Quentinchampenois) [inline config/locales/fr.yml:250]: Re add translations and keep rack_attack

File: config/secrets.yml [hunk 7]
```diff
@@ -20,9 +30,6 @@ default: &default
     enabled: <%= !ENV["HELP_SCOUT_BEACON_ID_MAIN"].blank? %>
     main: <%= ENV["HELP_SCOUT_BEACON_ID_MAIN"] %>
     fallback: <%= ENV["HELP_SCOUT_BEACON_ID_FALLBACK"] %>
```
Comment (Quentinchampenois) [inline config/secrets.yml:32]: Add hotjar

File: config/secrets.yml [hunk 8]
```diff
@@ -42,6 +49,12 @@ default: &default
       client_id: <%= ENV["OMNIAUTH_PUBLIK_CLIENT_ID"] %>
       client_secret: <%= ENV["OMNIAUTH_PUBLIK_CLIENT_SECRET"] %>
       site_url: <%= ENV["OMNIAUTH_PUBLIK_SITE_URL"] %>
+    france_connect:
```
Comment (Quentinchampenois) [inline config/secrets.yml:52]: Remove france_connect

File: config/secrets.yml [hunk 9]
```diff
@@ -52,6 +65,11 @@ default: &default
     server: <%= ENV["ETHERPAD_SERVER"] %>
     api_key: <%= ENV["ETHERPAD_API_KEY"] %>
     api_version: "1.2.1"
+  translator:
```
Comment (Quentinchampenois) [inline config/secrets.yml:68]: Remove translator

File: spec/system/confirmation_spec.rb [hunk 10]
```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require "spec_helper"
```
Comment (Quentinchampenois) [inline spec/system/confirmation_spec.rb:3]: Still needed ?

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]

**Review by paulinebessoles — APPROVED** [review]

**Pull request merged**